src/Mesh.cpp

```diff
@@ -153,21 +150,33 @@ std::vector<double> Mesh::getReactionRates(RxType rx) {
     }
 
     /* Tally the reaction rates summed across all groups */
+    double fsr_rx_rate = 0.;
```

Comment:
For optimization, only a boolean is needed here rather than a doule.

Reply:
So then add to the boolean and use implicit conversion ?

Reply:
My bad. I got it wrong.